deepcomp/env/single_ue/base.py

```diff
@@ -294,7 +296,8 @@ def step_reward(self, rewards):
         ue = self.ue_list[(self.time-1) % self.num_ue]
         return rewards[ue]
 
-    def done(self):
+    @staticmethod
+    def done():
```

Comment:
Not sure if that's a good idea. `done(self)` is overwritten in the child classes. What happens if it's declared as static in the parent class?

Reply:
Okay I'll revert the fix if you want but have a look at [this](https://deepsource.io/gh/withshubh/DeepCoMP/issue/PYL-R0201/description/).

Reply:
Ok thanks. I think the memory impact isn't that relevant here. Could you please revert it in this case?

In the other cases, the fixes to add `@staticmethod` look good to me and can stay. Thanks.

Reply:
Done! :sparkles: